game/static/game/css/rules.css

```
@@ -0,0 +1,366 @@
+    display: flex;
+    align-items: center;
+    gap: 12px;
+    text-decoration: none;
+}
+
+.logo-img {
+    height: 32px;
+    animation: logoGlow 3s ease-in-out infinite alternate;
+}
+
+@keyframes logoGlow {
```

Stylelint (17.11.0)

[error] Expected keyframe name "logoGlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)